portfolio/src/main/webapp/index.html

```diff
@@ -14,7 +14,7 @@
   <body>
     <!--NAV BAR-->
     <nav class="main-view">
-      <div id="menu-icon"><i class="fas fa-bars fa-2x"></i></div>
+      <div id="menu-icon" class="button"><i class="fas fa-bars fa-2x"></i></div>
```

Comment:
There should be a better class name for it

Reply:
Hmm... But I just want to use this class to style all buttons. Shouldn't I keep the name as general as possible then? :O Thanks in advance!

Reply:
I'm not sure if the css applied to this `button` and `icon` would be globally applicable, for example, right now in your css, the icon has fixed size etc, and I don't think that every icon in the website would be of the same size?

However, if that is indeed the case, this would be reasonable.